plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-orders-controller.php

```diff
@@ -118,6 +118,12 @@ protected function prepare_object_for_database( $request, $creating = false ) {
 
 			if ( ! is_null( $value ) ) {
 				switch ( $key ) {
+					case 'created_via':
```

Comment:
I am puzzled a bit by the removal of `'readonly' => true` in controllers V1 and V2 without replicating the changes from this file in those controllers. What's our thinking behind this approach?

Reply:
Oops, you are right, that didn't make sense, I've reverted the change on https://github.com/woocommerce/woocommerce/pull/57225/commits/084ddf6cc8675a84ac40e0f615e8d99c40ba7d09

Reply:
Looks good, thanks! We are good to merge.